checker/tests/calledmethods-lombok/BuilderTest.java

```diff
@@ -0,0 +1,126 @@
+// Generated by delombok at Thu May 16 14:44:28 PDT 2019
+
+import lombok.NonNull;
+
+public class BuilderTest {
```

Comment:
We should change these Lombok tests to run delombok on the original source rather than committing the delombok'd code.  We did this for returns receiver